gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-sse/src/main/java/io/gravitee/plugin/entrypoint/sse/SseEntrypointConnector.java

```diff
@@ -146,6 +184,10 @@ private Buffer errorToBuffer(Throwable error) {
     }
 
     private Buffer messageToBuffer(Message message) {
+        if (message == stoppingMessage) {
```

Comment:
.equals which doest first == comparaison but just to be safe in the futur.

```suggestion
        if (message.equals(stoppingMessage)) {
```